hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/TestUtils.java

```diff
@@ -475,7 +475,8 @@ public static void quasiCloseContainer(ContainerManagerV2 containerManager,
   public static StorageContainerManager getScmSimple(OzoneConfiguration conf)
       throws IOException, AuthenticationException {
     SCMConfigurator configurator = new SCMConfigurator();
-    configurator.setSCMHAManager(MockSCMHAManager.getInstance(true));
+    // TODO: TestDecommissionAndMaintenance will fail with this setting.
+//    conf.setBoolean(ScmConfigKeys.OZONE_SCM_HA_ENABLE_KEY, true);
```

Comment:
nit: indents before `//`

Reply:
done, after rebased from latest HDDS-2823 with fix of https://github.com/apache/ozone/pull/1745